dpbench/benchmarks/knn/__init__.py

```diff
@@ -5,11 +5,11 @@
 from .knn_initialize import initialize
 from .knn_numba_dpex_k import knn as knn_numba_dpex_k
 from .knn_numba_npr import knn as knn_numba_npr
-from .knn_numpy import knn as knn_numpy
+from .knn_python import knn as knn_python
 
 __all__ = [
     "initialize",
     "knn_numba_dpex_k",
     "knn_numba_npr",
-    "knn_numpy",
+    "knn_python",
```

Comment:
numpy is actually used here, in two places:
1. line 23: create empty matrix
2. line 86: deep copy (this one is going to removed after PR #136)

I think replacing line 23 to pure python will work.

Reply:
> numpy is actually used here, in two places:
> 
>     1. line 23: create empty matrix

It is fine to initialize arrays to `numpy.ndarray`. Python v/s numpy boils down to whether the computation is done through numpy library calls and vector style expressions or not. 

>     2. line 86: deep copy (this one is going to removed after #136)
> 
Yes, please update the implementation based on #136 that is now merged into main.
> 
> I think replacing line 23 to pure python will work.

Reply:
For line 23, I agree. But it might be better to remove it, so we can completely remove the numpy out of picture in this file. Also it can be confusing that python implementations has to call numpy without carefully checking the source code. What do you think?  @diptorupd

Reply:
Talked to Dipto. We will have numpy data allocation and init for python implementation. No change needed.